aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/states/AuthorizationState.kt

```diff
@@ -121,4 +147,14 @@ sealed class AuthorizationState : State {
             else -> authZState
         }
     }
+
+    class DeleteUserBuilder(private val authZState: AuthorizationState) :
```

Comment:
`DeleteUserBuilder` is not required as newState of `DeletingUser` can be created using the above builder.

Reply:
Removed